cookbooks/bcpc-hadoop/templates/default/hdp_hbase-regionserver-initd.erb

```diff
@@ -194,18 +194,26 @@ multi_hbase_daemon() {
         exit 1
     fi
 
-    export HBASE_${UPPERCASE_HBASE_DAEMON}_OPTS=" "
-
     for OFFSET in ${OFFSETS} ; do
+        export HBASE_${UPPERCASE_HBASE_DAEMON}_OPTS=" "
```

Comment:
Why even use `$HBASE_${UPPERCASE_HBASE_DAEMON}_OPTS` if it is just set to empty and not settable outside this script?